index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <script src="./script.js" defer></script>
+    <title>Zamalek official site</title>
+  </head>
+  <body class="fonts">
+    <header>
+      <nav class="topnav">
+        <div class="burger">
+          <a id="close-menu">&times;</a>
+          <ul class="burger-menu">
+            <li><a href="./about.html">About</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Videos</a></li>
+            <li><a href="#">Matches</a></li>
+            <li><a href="#">Teams</a></li>
+            <li><a href="#">Shop</a></li>
+          </ul>
+        </div>
+        <div class="navbars">
+          <ul class="nav1">
+            <li><a class="white" href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+            <li><a class="white" href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
+            <li><a class="white" href="#">English</a></li>
+            <li><a class="white" href="#">My Page</a></li>
+            <li><a class="white move" href="#">Logout</a></li>
+          </ul>
+          <div class="nav2">
+            <a class="logo-home" href="./index.html">
+            <img src="./images/icons/logo.png" alt="logo"/></a>
+            <ul>
+              <li><a href="./about.html">About</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Videos</a></li>
+              <li><a href="#">Matches</a></li>
+              <li><a href="#">Teams</a></li>
+              <li><a href="#">Shop</a></li>
+            </ul>
+            <a class="zamalek-fc">Zamalek FC</a>
+          </div>
+        </div>
+        <div class="icon">
+          <div class="vector"></div>
+          <div class="vector"></div>
+          <div class="vector"></div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="intro">
+        <p class="hello">"Hello! Zamalek fans"</p>
+        <h1>Election event for the most titled club in the 20th century.</h1>
+        <p class="header-description">
+          A general invitation to attend the Zamalek club elections for a new presidential term,
+          election festival , listen to candidates depates.
+        </p>
+        <p class="date">2022.11.10(THU) ~ 11(FRI)</p>
+        <p class="address">@ Cairo Nasser city zamalek club</p>
+      </section>
+      <section id="program" class="program">
```

Comment:
- [ Optional] The program cards are stretch vertically and there is vertical gap between the iconography, title and description.
- [Optional] There is an outline or border on mouse hover to the program cards in the design. Kindly implement
#### Design
![image](https://user-images.githubusercontent.com/44057863/169667506-ab0212d9-9ea2-46cb-9dce-ff853d130b6b.png)
#### Impl
![image](https://user-images.githubusercontent.com/44057863/169667525-fa36658e-56e6-4e54-ac33-0019076e86da.png)